config/utils/logger.js

```diff
@@ -0,0 +1,22 @@
+/* eslint no-console: 0 */
```

Comment:
prefer `/* eslint-disable no-console */`
Also good to know: `/* eslint no-console: 'warn' */` (useless here, but instead of 0 you can use 'off')